src/pkcs15init/profile.c

```diff
@@ -1985,24 +1985,40 @@ static struct block	root_ops = {
    "root",		process_block,	NULL,		root_blocks
 };
 
+static void
+get_inner_word(char *str, char word[64]) {
+	char *inner = NULL;
+	size_t len = 0;
+
+	inner = str;
+	while (isalnum((unsigned char)*inner) || *inner == '-' || *inner == '_') {
```

Comment:
This condition is already used in `__expr_get()` to filter the macro names. I would prefer to move the condition to separate function to have these two uses in sync and make sure the do not diverge.